README.md

```diff
@@ -30,7 +30,7 @@ Elchemy lets you write simple, fast and quality type safe code while leveraging
 - **Easy and type-safe interop**: You can call Elixir/Erlang without any extra boiler-plate. All the calls you make are checked in terms of type-safety as thoroughly as possible
 - **All the best of Elm and Elixir**: Elchemy inherits what's best in Elm - type safety, inference and extreme expressiveness, but also what's best in Elixir - Doc-tests, tooling and obviously the entire BEAM platform.
 - **Nearly No runtime errors** - Elchemy's type system **eliminates almost all runtime errors**. With a shrinking set of edge cases, your entire app will be as safe as the parts written in Elixir are.
-- **Beatiful and fully readable output** - All of the code produced by Elchemy can be easily read and analyzed without taking a single look at the source code
+- **Beautiful and fully readable output** - The produced code is idiomatic, performant and can be easily read and analyzed without taking a single look at the original source.
```

Comment:
@ShalokShalom opinions?

Reply:
Fine. The other PR is still open FYI

Reply:
Yup. Preparing a longer answer to that right now. It seems to touch much larger issue than just English semantics 😄